api/.air.toml

```diff
@@ -4,7 +4,7 @@ tmp_dir = "tmp"
 
 [build]
   args_bin = []
-  bin = "./bin/api"
+  bin = "./bin/api --stage development"
```

Comment:
Since we added some flags to productionize the API, we have to specify a development flag so that we can load .env when running locally.